+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Aug  8 00:52:23 2023 - INFO  - Start testing...
+ test_execution graphical.target
+ service=graphical.target
++ date '+%Y-%m-%d %T'
+ log_time='2023-08-08 00:52:23'
+ test_restart graphical.target
+ service=graphical.target
+ systemctl restart graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 127 0 0 'graphical.target restart failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='graphical.target restart failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'graphical.target restart failed'
+ LOG_ERROR 'graphical.target restart failed'
+ message='graphical.target restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target restart failed'
Tue Aug  8 00:52:23 2023 - ERROR - graphical.target restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Aug  8 00:52:24 2023 - ERROR - ../common/common_lib.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status graphical.target
+ grep 'Active: active'
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 1 0 0 'graphical.target restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='graphical.target restart failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'graphical.target restart failed'
+ LOG_ERROR 'graphical.target restart failed'
+ message='graphical.target restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target restart failed'
Tue Aug  8 00:52:30 2023 - ERROR - graphical.target restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Aug  8 00:52:31 2023 - ERROR - ../common/common_lib.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 127 0 0 'graphical.target stop failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='graphical.target stop failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'graphical.target stop failed'
+ LOG_ERROR 'graphical.target stop failed'
+ message='graphical.target stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target stop failed'
Tue Aug  8 00:52:31 2023 - ERROR - graphical.target stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Tue Aug  8 00:52:32 2023 - ERROR - ../common/common_lib.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status graphical.target
+ grep 'Active: inactive'
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 1 0 0 'graphical.target stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='graphical.target stop failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'graphical.target stop failed'
+ LOG_ERROR 'graphical.target stop failed'
+ message='graphical.target stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target stop failed'
Tue Aug  8 00:52:38 2023 - ERROR - graphical.target stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Tue Aug  8 00:52:38 2023 - ERROR - ../common/common_lib.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl start graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 127 0 0 'graphical.target start failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='graphical.target start failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'graphical.target start failed'
+ LOG_ERROR 'graphical.target start failed'
+ message='graphical.target start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target start failed'
Tue Aug  8 00:52:39 2023 - ERROR - graphical.target start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Aug  8 00:52:39 2023 - ERROR - ../common/common_lib.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status graphical.target
+ grep 'Active: active'
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ CHECK_RESULT 1 0 0 'graphical.target start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='graphical.target start failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'graphical.target start failed'
+ LOG_ERROR 'graphical.target start failed'
+ message='graphical.target start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'graphical.target start failed'
Tue Aug  8 00:52:46 2023 - ERROR - graphical.target start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Aug  8 00:52:46 2023 - ERROR - ../common/common_lib.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ test_enabled graphical.target
+ service=graphical.target
++ systemctl is-enabled graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Tue Aug  8 00:52:47 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2023-08-08 00:52:23' -u graphical.target
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of graphical.target'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of graphical.target'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload graphical.target
+ service=graphical.target
+ systemctl start graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ systemctl reload graphical.target
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit graphical.target'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit graphical.target'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit graphical.target'
+ LOG_ERROR 'Job type reload is not applicable for unit graphical.target'
+ message='Job type reload is not applicable for unit graphical.target'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit graphical.target'
Tue Aug  8 00:52:47 2023 - ERROR - Job type reload is not applicable for unit graphical.target
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Tue Aug  8 00:52:48 2023 - ERROR - ../common/common_lib.sh line 91
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl status graphical.target
+ grep 'Active: active'
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ systemctl status graphical.target
+ grep 'inactive (dead)'
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ return 1
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Aug  8 00:52:49 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Aug  8 00:52:49 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'Start environment cleanup.'
+ message='Start environment cleanup.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment cleanup.'
Tue Aug  8 00:52:50 2023 - INFO  - Start environment cleanup.
+ systemctl stop graphical.target
systemctl: error while loading shared libraries: libsystemd-shared-253.so: cannot open shared object file: No such file or directory
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue Aug  8 00:52:50 2023 - INFO  - Finish environment cleanup!
